components/pipeline-service/production/stone-prd-m01/deploy.yaml

```diff
@@ -1774,7 +1774,7 @@ spec:
             )" \
             --dry-run=client \
             -o yaml | kubectl apply -f -
-        image: quay.io/konflux-ci/appstudio-utils:ab6b0b8e40e440158e7288c73aff1cf83a2cc8a9@sha256:24179f0efd06c65d16868c2d7eb82573cce8e43533de6cea14fec3b7446e0b14
+        image: quay.io/redhat-appstudio/appstudio-utils:dbbdd82734232e6289e8fbae5b4c858481a7c057
```

Comment:
This line has to be reverted manually until we switch that line https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/production/base/kustomization.yaml#L11 to match the one in staging which points to a pipeline-service SHA including the fix.
The same goes for all production deploy.yaml files.